apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/FeatureExtractorSamples.kt

```diff
@@ -1,34 +1,456 @@
 package ch.tutteli.atrium.api.infix.en_GB.samples
 
-import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.because
+import ch.tutteli.atrium.api.infix.en_GB.extractSubject
+import ch.tutteli.atrium.api.infix.en_GB.feature
+import ch.tutteli.atrium.api.infix.en_GB.its
+import ch.tutteli.atrium.api.infix.en_GB.notToHaveElementsOrAll
+import ch.tutteli.atrium.api.infix.en_GB.of
+import ch.tutteli.atrium.api.infix.en_GB.toBeGreaterThan
+import ch.tutteli.atrium.api.infix.en_GB.toBeLessThan
+import ch.tutteli.atrium.api.infix.en_GB.toContain
+import ch.tutteli.atrium.api.infix.en_GB.toEndWith
+import ch.tutteli.atrium.api.infix.en_GB.toHaveElementsAndAll
+import ch.tutteli.atrium.api.infix.en_GB.toStartWith
+import ch.tutteli.atrium.api.infix.en_GB.withFailureDescription
 import ch.tutteli.atrium.api.verbs.expect
 import kotlin.test.Test
 
 class FeatureExtractorSamples {
+
     data class Person(
-        val firstName: String,
-        val lastName: String,
-        val age: Int,
-        val children: Collection<Person>
+        val name: String, val age: Int
+    )
+
+    @Test
+    fun itsFeature() {
+        val person = Person(name = "John Smith", age = 25)
+
+        expect(person) its { age } toBeLessThan 30 toBeGreaterThan 20 // subject is now Int
+
+
+        fails {
+            expect(person) its { age } toBeLessThan 20 toBeGreaterThan 30 // subject is now Int; fails
```

Comment:
Can you please reformat to something like the following (otherwise it is hard to know to what a comment belongs (see also CollectionFeatureExtractorSamples):

```suggestion
            expect(person) its { age } toBeLessThan 20 toBeGreaterThan 30
            //             |           |                | not reported because `toBeLessThan` already fails
            //             |           | fails
            //             | subject is now Int
```

Reply:
Reformatted ✅